lrc_file/LrcFile.py

```diff
@@ -65,7 +56,7 @@ def _get_flow_metrics(lnst_run: RecipeRun, evaluated_flow_metrics: list[str]) ->
 
 def _get_cpu_metrics(lnst_run: RecipeRun, evaluated_cpu_metrics: list[str]) -> dict[str, float]:
     return {
-        f"{i}_{_get_cpu_hostid(result)}": value.average
+        f"{i}_utilization": value.average
```

Comment:
i think this may be the `key` from the deeper for loop, or there may be a more programatic way to get the `utilization` string instead of hard coding it. can you quickly check?

Reply:
`key` is name of (sub)measurement - `cpu`, `cpu1`, `...`, `cpuN`.  
actually lnst uses `utilization` during evaluation only. so there is a way but complicated and hacky as i would need to find evaluation result in the results, somehow get `cpu` evaluations only and parse it from there.